### PR TITLE
Fix markdown mistakes

### PR DESCRIPTION
**Pull request by garazdawi**
I was looking through all EEPs and found a couple of markdown mistakes.

### CODE REVIEW

File: eeps/eep-0055.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ Abstract
 
 This EEP proposes the addition of a new unary operator `^` for
 explicitly marking variables in patterns as being already bound.  This
-is known as "pinning" in Elixir - see [Elixir doc][the Elixir
-documentation].
+is known as "pinning" in Elixir - see (the Elixir documentation)[Elixir doc].
```
Comment (zuiderkwast) [inline eeps/eep-0055.md:17]:
```suggestion
is known as "pinning" in Elixir - see [the Elixir documentation][Elixir doc].
```

File: eeps/eep-0012.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ there are four possible outcomes:
 4. Expr yields true => the test succeeds;
    this might as well have been Expr without Pattern,
    and Pattern = true beforehand.
-======================================================================
+
```
Comment (zuiderkwast) [inline eeps/eep-0012.md:78]: (I can't comment on lines outside the diff.) A blank line is missing before the numbered list above. The list is currently broken. (See https://erlang.org/eeps/eep-0012.html)

There are more blank lines missing in this EEP, for example before and after every code block in the next section.

**Review by zuiderkwast — COMMENTED** [review]
Nice!

File: eeps/eep-0007.md [hunk 3]
```diff
@@ -740,7 +741,7 @@ are (probably) going to be introduced in Erlang.
 
 Thus, the high-level FFI API should be considered highly experimental
 and subject to change, depending on how type contracts will allow to
-achieve the same tasks (see [High-level API][]).  This issue will need
+achieve the same tasks (see [High-level API]).  This issue will need
```
Comment (zuiderkwast) [inline eeps/eep-0007.md:744]: Did you test this? The original Markdown spec allows this link syntax: https://daringfireball.net/projects/markdown/syntax#link. It's shorthand for `[High-level API][High-level API]`. I'm not sure `[High-level API]` becomes a link.

**Comment by paulo-ferraz-oliveira** [comment]
@garazdawi, it is possible to add some kind of Markdown validation to CI, via `npm`, for example. Let me know if you're interested and I might try to find time to tackle it. It's done for `setup-beam` FWIW.

File: eeps/eep-0007.md (hunk 3 above)
Comment (garazdawi) [reply eeps/eep-0007.md]: I've removed the link alltogether.

**Comment by garazdawi** [comment]
markdownlint reports 2940 warnings in these markdown files :)

```
      1 MD031/blanks-around-fences
      1 MD034/no-bare-urls
      1 MD047/single-trailing-newline
      2 MD027/no-multiple-space-blockquote
      2 MD037/no-space-in-emphasis
      3 MD040/fenced-code-language
      3 MD046/code-block-style
      4 MD028/no-blanks-blockquote
      4 MD038/no-space-in-code
      5 MD033/no-inline-html
      5 MD036/no-emphasis-as-heading/no-emphasis-as-header
      7 MD024/no-duplicate-heading/no-duplicate-header
      7 MD026/no-trailing-punctuation
      7 MD029/ol-prefix
     13 MD001/heading-increment/header-increment
     22 MD032/blanks-around-lists
     57 MD041/first-line-heading/first-line-h1
     65 MD022/blanks-around-headings/blanks-around-headers
    122 MD007/ul-indent
    142 MD004/ul-style
    250 MD013/line-length
    256 MD003/heading-style/header-style
    258 MD010/no-hard-tabs
    271 MD030/list-marker-space
    416 MD009/no-trailing-spaces
   1016 MD012/no-multiple-blanks
```

If you would like to spend the time fixing them that would be most welcome :) Seems like most of them are space/newline related.

**Comment by paulo-ferraz-oliveira** [comment]
Oh, yeah, but you can configure this stuff so that some errors/warnings are not considered.

**Pull request merged**